aspnetcore/tutorials/first-web-api.md

```diff
@@ -294,6 +294,13 @@ The generated code:
 * Marks the class with the [`[ApiController]`](/dotnet/api/microsoft.aspnetcore.mvc.apicontrollerattribute) attribute. This attribute indicates that the controller responds to web API requests. For information about specific behaviors that the attribute enables, see <xref:web-api/index>.
 * Uses DI to inject the database context (`TodoContext`) into the controller. The database context is used in each of the [CRUD](https://wikipedia.org/wiki/Create,_read,_update_and_delete) methods in the controller.
 
+The ASP.NET Core templates for:
+
+* Controllers with views include `[action]` in the route template.
+* API controllers don't include `[action]` in the route template.
+
+When `[action]` isn't in the route template, the [action](#action) name is not in the route. That is, the method name isn't used in the matching route.
```

Comment:
Suggestion to avoid the double negative and to further clarify in the 2nd sentence:

```suggestion
When the `[action]` token isn't in the route template, the [action](#action) name is excluded from the route. That is, the action's associated method name isn't used in the matching route.
```